misc/io.github.Tremulous.appdata.xml

```diff
@@ -0,0 +1,61 @@
+  <summary>Aliens vs Humans, First Person Shooter game with elements of Real Time Strategy</summary>
+  <description>
+    <p>
+      Tremulous is a free, open source game that blends a team based FPS with
+      elements of an RTS.
+    </p>
+    <p>
+      Players can choose from 2 unique races, aliens and humans.
+      Players on both teams are able to build working structures in-game like an
+      RTS.
+    </p>
+  </description>
+  <url type="homepage">https://grangerhub.com</url>
+  <screenshots>
+    <screenshot type="default"><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/a.png</image></screenshot>
+    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/b.png</image></screenshot>
+    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/c.png</image></screenshot>
+    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/d.png</image></screenshot>
+    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/e.png</image></screenshot>
+  </screenshots>
+  <update_contact>mail@asciiwolf.com</update_contact>
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="violence-realistic">moderate</content_attribute>
+    <content_attribute id="violence-bloodshed">mild</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
+  <releases>
+    <release version="1.3.0-alpha.0.14" date="2019-09-14"/>
+    <release version="1.3.0-alpha.0.13" date="2018-02-23"/>
+    <release version="1.3.0-alpha.0.12" date="2017-10-19"/>
+  </releases>
```

Comment:
Please note that the releases section is useful only if it is actively updated before every GrangerHub Tremulous release. If it is outdated, it is useless and problematic, and is better to remove it - the Flathub AppStream parser is as far as I know the only one that requires the releases section (and it can be added as a downstream patch).

Reply:
Removed.